concepts/Common/Accessibility/00 Overview/35 Standard Compliance.md

```diff
@@ -0,0 +1,5 @@
+Compliance with the <a href="https://www.w3.org/WAI/standards-guidelines/wcag/" target="_blank">Web Content Accessibility Guidelines (WCAG)</a> and <a href="https://www.access-board.gov/ict/" target="_blank">Section 508</a> standards is essential for DevExtreme components to prioritize accessibility.
```

Comment:
Compliance is essential for DevExtreme components to prioritize accessibility? Can you explain what you mean here? Do you mean DevExtreme components prioritize accessibility?

Reply:
Yes, I mean that

Reply:
How about 

```suggestion
DevExtreme components prioritize accessibility and comply with the <a href="https://www.w3.org/WAI/standards-guidelines/wcag/" target="_blank">Web Content Accessibility Guidelines (WCAG)</a> and <a href="https://www.access-board.gov/ict/" target="_blank">Section 508</a> standards.
```

or

```suggestion
Compliance with the <a href="https://www.w3.org/WAI/standards-guidelines/wcag/" target="_blank">Web Content Accessibility Guidelines (WCAG)</a> and <a href="https://www.access-board.gov/ict/" target="_blank">Section 508</a> standards ensure DevExtreme components prioritize accessibility.
```

or

```suggestion
DevExtreme components prioritize accessibility by complying with the <a href="https://www.w3.org/WAI/standards-guidelines/wcag/" target="_blank">Web Content Accessibility Guidelines (WCAG)</a> and <a href="https://www.access-board.gov/ict/" target="_blank">Section 508</a> standards.
```